tests/unit/backend/corpora/dataset_processing/test_h5ad_data_file.py

```diff
@@ -174,6 +174,28 @@ def test__to_cxg__with_sparse_column_encoding(self):
         # Clean up
         remove(sparse_with_column_shift_filename)
 
+    def test__slash_in_attribute_name(self):
+        # tiledb failure repro code from https://github.com/TileDB-Inc/TileDB-Py/issues/294
+        # this fails (throws a TileDBError) for 0.8.0 and below but works for 0.8.1+
+        import numpy as np
```

Comment:
Quick question:
Is there a general practice reason for importing within the test vs. in the module? These imports already exist on lines 8 and 10.

Reply:
Ah, good point! Copy pasta 😄

Reply:
Fixed